docs/UserGuide/Cluster/Cluster-Setup.md

```diff
@@ -143,284 +134,212 @@ Please set the important parameters in iotdb-datanode.properties:
 | dn\_schema\_region\_consensus\_port | Schema replicas communication port for consensus |
 | dn\_target\_config\_node\_list      | Running ConfigNode of the Cluster                |
 
-Start on Linux:
-```
-# Foreground
-bash ./sbin/start-datanode.sh
+**Notice: The preceding configuration parameters cannot be changed after the node is started. Ensure that all ports are not occupied. Otherwise, the Node cannot be started.**
 
-# Background
-nohup bash ./sbin/start-datanode.sh >/dev/null 2>&1 &
-```
+# 6. Cluster Operation
 
-Start on Windows:
-```
-sbin\start-datanode.bat
-```
+## 6.1 Starting the cluster
 
-More details are in [DataNode Configurations](https://iotdb.apache.org/UserGuide/Master/Reference/DataNode-Config-Manual.html).
+This section describes how to start a cluster that includes several ConfigNodes and DataNodes.
+The cluster can provide services only by starting at least one ConfigNode
+and no less than the number of data/schema_replication_factor DataNodes.
 
-### Stop IoTDB
-When you meet problem, and want to stop IoTDB ConfigNode and DataNode directly, our shells can help you do this.
+The total process are three steps:
 
-In Windows:
+* Start the Seed-ConfigNode
+* Add ConfigNode (Optional)
+* Add DataNode
 
-```
-sbin\stop-datanode.bat
-```
-```
-sbin\stop-confignode.bat
-```
-In Linux:
-```
-bash sbin/stop-datanode.sh
-```
-```
-bash sbin/stop-confignode.sh
-```
-Be careful not to miss the "sudo" label, because some port info's acquisition may require root authority. If you can't sudo, just
-use "jps" or "ps aux | grep iotdb" to get the process's id, then use "kill -9 <process-id>" to stop the process.  
+### 6.1.1 Start the Seed-ConfigNode
 
-## Start StandAlone
-If you just want to setup your IoTDB locally, 
-You can quickly init 1C1D (i.e. 1 Confignode and 1 Datanode) environment by our shells.
+The first ConfigNode to start is the Seed-ConfigNode, which marks the creation of the new cluster.
```

Comment:
before users reading this chapter, all the settings are done! 
so remove current content and add:

Please make sure XXXXX is set correct in XXXXX. Suppose server A is set to the seed-configNode, then login server A and execute the following commands:

Reply:
Fixed~